### ISSUE TITLE
Combining losses for Tensorboard

### ISSUE THREAD

**Issue by SreenivasVRao** [matterport/Mask_RCNN#860]
Is it possible to combine the val_loss and loss graphs and display them in tensorboard? I know you can do it with tf.summary, but I'm not sure how to do it with Keras callbacks.

**Comment by SreenivasVRao** [comment]
https://stackoverflow.com/questions/47877475/keras-tensorboard-plot-train-and-validation-scalars-in-a-same-figure/48393723

I found this after some digging around.
